system/db/chatmacros.txt

```diff
@@ -0,0 +1,16 @@
+﻿// Melia
+// Database file
+//---------------------------------------------------------------------------
+
+[
+{ "id": "1", "name": "Set1", "pose": "8", "text": "@dicID_^*$ETC_20151001_014770$*^" },
```

Comment:
This suffers from the same issue as #103.

Reply:
Indeed =/ Having values like these seems so pointless, because you have *no* idea what the actual text is until you look it up.

Reply:
It's true. The alternative in this case would be something like `{img emoticon_0001 30 30}`. Do you think this is better in this scenario?

Reply:
Update:
I realized that this is not data that we need to keep in Melia (at least for now).

Reply:
Well, if it were the official value (`{img emoticon_0022 30 30} I'm sorry`), at the very least a user could see that, and change it if they wanted to do. The way it is now it's just some cryptic dict id, where a user might not even think they *can* change it, and if so, to what.

Reply:
All good points. The client will send the `dicID` if I recall. What do you think about converting this value to regular text before it is stored in the database?

Reply:
The client sends the dicID? Isn't the text customizable? Or do you mean it sets up and sends the default?